pkgs/misc/uboot/default.nix

```diff
@@ -136,6 +136,14 @@ in {
     dontStrip = false;
     extraMeta.platforms = lib.platforms.linux;
     extraMakeFlags = [ "HOST_TOOLS_ALL=y" "CROSS_BUILD_TOOLS=1" "NO_SDL=1" "tools" ];
+
+    outputs = [ "out" "man" ];
+
+    postInstall = ''
+      mkdir -p $man/share/man/man1
+      cp doc/*.1 $man/share/man/man1
+      gzip $man/share/man/man1/*.1
```

Comment:
Even without my suggestion, you don't need to run `gzip` yourself, `stdenv` does it by itself.